pkg/kv/kvserver/kvflowcontrol/rac2/range_controller.go

```diff
@@ -2857,14 +2911,22 @@ func (rss *replicaSendStream) tryHandleModeChangeRaftMuAndStreamLocked(
 	}
 }
 
+func (rss *replicaSendStream) exceedInflightBytesThresholdRaftMuAndStreamLocked() bool {
+	rss.parent.parent.opts.ReplicaMutexAsserter.RaftMuAssertHeld()
+	rss.mu.AssertHeld()
+	return rss.mu.inflightBytes > rss.parent.parent.opts.RaftMaxInflightBytes
```

Comment:
`>=` ? This is minor, but seems logical to pause the stream in the case we're exactly at the limit. This would match the raft's [heuristic](https://github.com/cockroachdb/cockroach/blob/b7331e4abaa1e9b10065c4e3b43b29c56dcb7f1d/pkg/raft/tracker/inflights.go#L102).